src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -74,6 +74,10 @@ export type Context = {
    * The ID for the current scenario test being run.
    */
   id: string
+  /**
+   * Which step of setup we are on
+   */
+  step?: 'database' | 'scenario'
```

Comment:
This is temporary for now. This exists so we can connect to different databases in MSSQL tests. We need to explicitly connect to different databases because MSSQL does not support switching DBs on the fly (the `mssql` package doesn't)